native/cocos/ui/webview/WebViewImpl-openharmony.cpp

```diff
@@ -130,6 +129,9 @@ void WebViewImpl::loadURL(const ccstd::string &url) {
     vals.insert(std::make_pair("tag", _viewTag));
     vals.insert(std::make_pair("url", url));
     NapiHelper::postMessageToUIThread("loadUrl", vals);
+
+    //canGoBack();
+    //canGoForward();
```

Comment:
As the implementation is comment, so i think don't have to comment these codes too.

Reply:
This is debugging code, in order to verify part of the function, I forgot to delete